features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2204,7 +2204,9 @@ CREATE TABLE IDN_ACTION (
 CREATE TABLE IDN_ACTION_PROPERTIES (
     ACTION_UUID CHAR(36) NOT NULL,
     PROPERTY_NAME VARCHAR(100) NOT NULL,
-    PROPERTY_VALUE VARCHAR(255) NOT NULL,
+    PROPERTY_TYPE VARCHAR(9) NOT NULL DEFAULT 'PRIMITIVE',
```

Comment:
For this schema is default value is needed ? IMO, we can just keep it not null so that the application should set it.
In product migration schema changes we have to alter the table with the default

Reply:
Lets have the default value, until application logic supports to set the value. 
we can remove the default value with next BE changes. This ensures that current on prem and asgardeo schemas are identical.